packages/next/package.json

```diff
@@ -73,7 +73,7 @@
   ],
   "dependencies": {},
   "peerDependencies": {
-    "@tanstack/react-query": "^5.59.15",
+    "@tanstack/react-query": "^5.54.1",
```

Comment:
keeping the peer dep as-is for now

will bump it as part of #5913 where it's actually required.